### DISCUSSION TITLE
precision of statistics card

### DISCUSSION THREAD

**Discussion by Appie-rappie** [home-assistant/frontend#17268] [Dashboard] [answered]
The data in a statistics card is rounded to 2 decimals and not configurable. While this is acceptable in the majority of situations, it is a major nuisance in specific cases, such as in a calculation of the average particle size, which typically varies between 0.45 and 0,48.
The problem is illustrated by the two graphs below, one a history graph, the other a statistics graph, both showing the exact same data and time window. 
![Capture](https://github.com/home-assistant/frontend/assets/68786808/7daedfa2-d5f4-4f76-8691-cee57f61ba83)
Why is the data rounded in a statistics card and if this is needed, why can't it be set to a different precision by the user?

**Comment by OZ1SEJ** [comment]
If it isn't already, this really should be created as an issue.

**Comment by karwosts** [comment, marked as answer]
Fixed last year with https://github.com/home-assistant/frontend/pull/18312
